rosa_hcp/rosa-hcp-sts-creating-a-cluster-quickly.adoc

```diff
@@ -79,11 +79,18 @@ You must have a Virtual Private Cloud (VPC) to create {hcp-title} cluster. You c
 
 [NOTE]
 ====
-The Terraform instructions are for testing and demonstration purposes. Your own installation requires some modifications to the VPC for your own use.
+The Terraform instructions are for testing and demonstration purposes. Your own installation requires some modifications to the VPC for your own use. You should also ensure that when you use this Terraform script it is in the same region that you intend to install your cluster. In these examples, use `us-east-2`.
 ====
 
 [discrete]
 include::modules/rosa-hcp-vpc-terraform.adoc[leveloffset=+3]
+
+[role="_additional-resources"]
```

Comment:
Based on the Content Checklist [1] > Additional resources specific requirements:

1. I do not believe you need the id.
2. We cannot use paragraph text as part of the xref.
3. Also - the guidance is to avoid linking to GitHub. Per the peer review guidelines, an exception would have to be granted by your Product Experience team rep. 

[1] https://docs.engineering.redhat.com/display/JUP/Jupiter+Source+files+readiness+Checklist#JupiterSourcefilesreadinessChecklist-ContentChecklist

Reply:
@mjpytlak That checklist covers requirements for a deprecated tool; the guidance in the contribution guidelines only refer to unordered lists rather than paragraphs, and do not specify that link text only should be used: https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc?plain=1#L1699

Reply:
@laubai OK. Thanks for pointing that out.

Reply:
@mjpytlak We'll also make an exception for the Github linking since this is a new offering (a Terraform provider that we created specifically for creating VPCs). We have another project that is documenting [our Terraform provider](https://github.com/terraform-redhat/terraform-provider-ocm), so this is new territory.